cmd/da-light-client/start/start.go

```diff
@@ -50,7 +50,7 @@ func Cmd() *cobra.Command {
 }
 
 func addFlags(cmd *cobra.Command) {
-	cmd.Flags().StringP(rpcEndpointFlag, "", "", "The DA rpc endpoint to connect to.")
+	cmd.Flags().StringP(rpcEndpointFlag, "", celestia.DefaultCelestiaRPC, "The DA rpc endpoint to connect to.")
```

Comment:
Why u set celestia's consts in the main command?

In the Celestia implementation, if no RPC is set, it should take the default.
This const should be defined in the celestia package